### PR TITLE
add display-name and help-text for idp-email-verification and idp-username-password

### LINKED ISSUE #12055: Missing messages for IdpUsernamePassword and IdpEmailVerification authenticators while listing tryAnotherWay options
### Describe the bug

In the first broker login, when clicking "Try another way" while verifying accounts existance, the display names and help texts of IdpUsernamePassword and IdpEmailVerification are not displayed, with the keys being displayed instead.

We fixed it on an older versions on our side, and since it seems this has not been fixed here yet, I'll submit a PR in a minute.

### Version

16.1.0

### Expected behavior

A human readable value should be displayed in the HTML.

Example for IdpEmailVerification : "Email Verification" and "Confirm the link by validating your email."

### Actual behavior

Only the keys are displayed.

Example for IdpEmailVerification : "idp-email-verification-display-name" and "idp-email-verification-help-text"

### How to Reproduce?

_No response_

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by Crow-EH**
Closes #12055

### CODE REVIEW

File: themes/src/main/resources-community/theme/base/login/messages/messages_fr.properties [hunk 1]
```diff
@@ -281,3 +281,9 @@ noCertificate=[Pas de certificat]
 
 pageNotFound=Page non trouv\u00e9e
 internalServerError=Une erreur interne du serveur s''est produite
+
+# Identity provider
+idp-email-verification-display-name=Vérification du courriel
```
Comment (FranckKe) [inline themes/src/main/resources-community/theme/base/login/messages/messages_fr.properties:286]: I did not double check if unicode is really required but as the rest of translations are using it:
```suggestion
idp-email-verification-display-name=V\u00e9rification du courriel
```

File: themes/src/main/resources-community/theme/base/login/messages/messages_fr.properties [hunk 2]
```diff
@@ -281,3 +281,9 @@ noCertificate=[Pas de certificat]
...
+idp-email-verification-help-text=Lier votre compte en validant votre courriel.
+idp-username-password-form-display-name=Nom d''utilisateur et mot de passe
+idp-username-password-form-help-text=Lier votre compte en vous connectant.
```
Comment (FranckKe) [inline themes/src/main/resources-community/theme/base/login/messages/messages_fr.properties:289]: The translation is slightly different than the English source. It should work but just to double check if that is what you meant: `Link your account by login in.`

Comment (Crow-EH) [reply themes/src/main/resources-community/theme/base/login/messages/messages_fr.properties:289]: Yup, that's exactly it.

When login in with an  external identity provider, if you already have an account on our keycloak (same email on both side in our case), then you can try to login on our keycloak to prove you own it before linking it to the idp account.

**Comment by Crow-EH** [comment]
Thank you for your review @FranckKe

File: themes/src/main/resources-community/theme/base/login/messages/messages_fr.properties (hunk 2 above)
Comment (Crow-EH) [reply themes/src/main/resources-community/theme/base/login/messages/messages_fr.properties:289]: + `Link your account by validating your email.`

File: themes/src/main/resources-community/theme/base/login/messages/messages_fr.properties (hunk 1 above)
Comment (Crow-EH) [reply themes/src/main/resources-community/theme/base/login/messages/messages_fr.properties]: done

File: themes/src/main/resources/theme/base/login/messages/messages_en.properties [hunk 3]
```diff
@@ -474,6 +474,10 @@ webauthn-error-user-not-found=Unknown user authenticated by the Security key.
 # Identity provider
 identity-provider-redirector=Connect with another Identity Provider
 identity-provider-login-label=Or sign in with
+idp-email-verification-display-name=Email Verification
```
Comment (FranckKe) [inline themes/src/main/resources/theme/base/login/messages/messages_en.properties:477]: What is the motive behind capitalizing `Verification`?

Comment (FranckKe) [reply themes/src/main/resources/theme/base/login/messages/messages_en.properties:477]: Never mind, I just saw that `Verification` is also capitalized in different other translations.

**Review by ssilvert — DISMISSED** [review]

**Comment by Crow-EH** [comment]
This will conflict with https://github.com/keycloak/keycloak/pull/12054, so I'll rebase once it's merged

**Review by ssilvert — APPROVED** [review]

**Pull request merged**